types/normalize-port/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for normalize-port 1.16
+// Project: git@github.com:aoberoi/normalize-port.js.git
+// Definitions by: Alex Alonso <https://github.com/finaldes>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare module "normalize-port" {
```

Comment:
You should still remove the `declare module` and just directly `export = normalizePort;` on the top-level.